api/backend/app/controllers/auth_router.py

```diff
@@ -0,0 +1,61 @@
+from classy_fastapi import Routable, post, get
+from pydantic import BaseModel
+from backend.app.utils.signed_data import verify_signed_data, extract_signed_address_from_signature_header
+from backend.app.services.user_service import UserService
+from backend.app.auth.jwt_token import generate_jwt_token_using_user_address
+from fastapi.responses import JSONResponse
+from fastapi import Response, HTTPException, Request
+from backend.app.models.user.response_dto import UserResponse
+from backend.app.models.user.user_dto import SignatureDataDto, UserCreateDto
+import os
+
+
+class AuthRouter(Routable):
+
+    def __init__(self, user_service: UserService = UserService(), *args, **kwargs) -> None:
+        super().__init__(*args, **kwargs)
+        self.user_service = user_service
+        current_environment = os.environ.get("NODE_ENV")
+        self.secure = True if current_environment == "production" else False
+        self.domain = "api.agents.cardanoapi.io" if current_environment == "production" else "localhost"
+
+    @post("/login")
```

Comment:
set base path for all auth endpoint to start with '/auth'

Reply:
Set base path '/auth'  in both frontend and backend.